dbtmetabase/parsers/dbt_folder.py

```diff
@@ -144,6 +144,9 @@ def _read_model(
             metabase_columns.append(self._read_column(column, schema))
 
         description = model.get("description", "")
+        meta = model.get("meta")
```

Comment:
If you retrieve it as `model.get("meta", {})`, you don't need to do ternaries for each field below.